torch/_inductor/triton_heuristics.py

```diff
@@ -50,11 +50,6 @@
     KernelInterface = object
     OutOfResources = object
 
-if has_triton():
```

Comment:
Why do we need to remove the check?

Reply:
> Why do we need to remove the check?
Here they use `get_cuda_stream` from [triton](https://github.com/openai/triton/blob/v2.1.0/python/triton/runtime/jit.py#L20C31-L20C31)
Talked with guangye, the get_cuda_stream in triton is a workaround for compatibilitythe old version that `_cuda_getCurrentRawStream` has not implemented. 
Currently we can directly use `_cuda_getCurrentRawStream`.